frontend/src/features/properties/map/acquisition/add/AddAcquisitionFileYupSchema.ts

```diff
@@ -0,0 +1,10 @@
+/* eslint-disable no-template-curly-in-string */
```

Comment:
need to disable this lint rule to be able to parameterize yup validation string below. 

Perhaps we should discuss adding a project-wide exception for yup files